kolibri/core/content/management/commands/importcontent.py

```diff
@@ -565,6 +630,16 @@ def _start_file_transfer(self, f, filetransfer):
         return FILE_TRANSFERRED, data_transferred
 
     def handle_async(self, *args, **options):
+        if not options["directory"] and not options["manifest"]:
+            raise CommandError(
+                "Either a directory or a manifest file must be provided."
+            )
```

Comment:
This makes impossible to import from network without a manifest. And there's an exception because `options["directory"]` is not an existing option when importing from network

Reply:
Good spotting! That's fixed in the newest push.